_posts/2023-02-07-homebrew-4.0.0.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 4.0.0
+author: MikeMcQuaid
+redirect_from: /blog/4.0.0/
+---
+
+Today I'd like to announce Homebrew 4.0.0. The most significant change since 3.6.0 is faster updating through JSON downloads.
```

Comment:
```suggestion
Today, I'd like to announce Homebrew 4.0.0. The most significant change since 3.6.0 enables significantly faster Homebrew-maintained tap updates by migrating from git-cloned taps to JSON downloads.
```